### DISCUSSION TITLE
MeC ArchiMate

### DISCUSSION THREAD

**Discussion by vince-fabmob** [VilledeMontreal/MontrealEnCommun#16] [General]
Salut @houssamkhalifeh , est-ce possible d'avoir une image plus détaillée? Je ne réussi pas à lire le shéma. 

https://github.com/VilledeMontreal/MontrealEnCommun/blob/master/R%C3%A9f%C3%A9rentiel/Archi/MeC-ArchiMate.PNG

**Comment by houssamkhalifeh** [comment]
Salut @vince-fabmob, l'image est pour donner une idée de l'interface Archi. L'idéal est de télécharger Archi et ouvrir le diagramme directement dans Archi avec tous les autres digrammes du modèle MeC. 

1. Télécharger Archi à partir du lien https://www.archimatetool.com/
2. Télécharger le fichier du modèle MeC [MeC.archimate](https://github.com/VilledeMontreal/MontrealEnCommun/blob/master/R%C3%A9f%C3%A9rentiel/Archi/MeC.archimate)
3. Ovrir le fichier avec Archi, visualiser et modifier.

**Reply by houssamkhalifeh** [reply]
Salut @vince-fabmob, J'ai ajouté une image de l'état actuel du modèle au cas où.
https://github.com/VilledeMontreal/MontrealEnCommun/blob/master/R%C3%A9f%C3%A9rentiel/Archi/Montr%C3%A9al%20en%20commun.png
Mais, comme le modèle va évoluer, l'image devrait ètre reproduite fréquement pour la garder en sync.

**Reply by vince-fabmob** [reply]
Ok, je crois que je dois demander aux TI pour l'installation.
